libraries/c_sdk/standard/ble/include/iot_ble_mqtt_transport.h

```diff
@@ -66,19 +84,19 @@ MQTTStatus_t IotBleMqttTransportAcceptData( const NetworkContext_t * pContext );
 /**
  * @brief Transport interface write function.
  *
- * @param[in] context An opaque used by transport interface.
+ * @param[in] pContext An opaque used by transport interface.
  * @param[in] buf A pointer to a buffer containing data to be sent out.
  * @param[in] bytesToWrite number of bytes to write from the buffer.
  * @return the number of bytes sent.
  */
 int32_t IotBleMqttTransportSend( NetworkContext_t * pContext,
```

Comment:
The transport send will have to match the type of the updated transport interface definition in here
https://github.com/FreeRTOS/coreMQTT/blob/master/source/portable/transport_interface.h#L185, otherwise certain platforms will fail build.

Reply:
Same comment for the receive as well.

Reply:
Updated the `send` and `receive` function signatures.

Reply:
Thanks